src/library_fs.js

```diff
@@ -1221,9 +1219,9 @@ mergeInto(LibraryManager.library, {
       if (typeof data === 'string') {
         var buf = new Uint8Array(lengthBytesUTF8(data)+1);
         var actualNumBytes = stringToUTF8Array(data, buf, 0, buf.length);
-        FS.write(stream, buf, 0, actualNumBytes, 0, opts.canOwn);
+        FS.write(stream, buf, 0, actualNumBytes, void 0, opts.canOwn);
```

Comment:
better to just write `undefined`, I think, the `void` operator is weird :)

Reply:
Or `null` may be better, depending on the context - I'm not sure?

Reply:
Both are fine here. It's safer to use `void 0` as `undefined` can be a global variable name and `void` is a keyword, but yes, normally we don't have to care.